src/js/components/Button/__tests__/Button-test.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import renderer from 'react-test-renderer';
 import { cleanup, fireEvent, render } from '@testing-library/react';
 import 'jest-styled-components';
+// import { Add, Next } from 'grommet-icons';
```

Comment:
Clean up commented out code?

Reply:
I've put it back in as I added back the tests.